Project.toml

```
@@ -3,6 +3,7 @@ uuid = "efcf1570-3423-57d1-acb7-fd33fddbac46"
 version = "0.4.1"
```

```suggestion
 version = "0.5.0"
```

This is backward incompatible so we need to bump the leading version.